skiko/src/awtMain/objectiveC/macos/MetalContextHandler.mm

```diff
@@ -0,0 +1,64 @@
+#ifdef SK_METAL
+
+#import <jawt.h>
+#import <jawt_md.h>
+
+#import <QuartzCore/CAMetalLayer.h>
+#import <Metal/Metal.h>
+#import <GrDirectContext.h>
+#import <mtl/GrMtlBackendContext.h>
+#import <mtl/GrMtlTypes.h>
+
+#import "MetalDevice.h"
+
+extern "C"
+{
+JNIEXPORT jlong JNICALL Java_org_jetbrains_skiko_context_MetalContextHandler_makeMetalContext(
+    JNIEnv* env, jobject contextHandler, jlong devicePtr)
+{
+    @autoreleasepool {
```

Comment:
I do not see the reason for `autoreleasepool` here. Usually you need to put it explicitly inside loop or so, but not sure why you need it here

See https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/MemoryMgmt/Articles/mmAutoreleasePools.html

Reply:
Ok, keeping it here is safe, I meant it can be just redundant

Reply:
BTW, from the link that you mentioned: 
> Cocoa always expects code to be executed within an autorelease pool block, otherwise autoreleased objects do not get released and your application leaks memory.

Reply:
Yep, this is why it usually placed in `main` function. Not in every function like here.